build.csx

```diff
@@ -16,7 +16,7 @@ var resharperCltUrl = new Uri("https://download.jetbrains.com/resharper/JetBrain
 var resharperCltPath = $"{Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData)}/.resharper/{resharperCltUrl.Segments.Last()}";
 var inspectCodePath = $"./.resharper/{Path.GetFileNameWithoutExtension(resharperCltUrl.Segments.Last())}/inspectcode.exe";
 var nuget = "src/.nuget/v4.0.0/NuGet.exe";
-var msBuild = $"{Environment.GetFolderPath(Environment.SpecialFolder.ProgramFilesX86)}/MSBuild/14.0/Bin/msbuild.exe";
+var msBuild = $"{Environment.GetFolderPath(Environment.SpecialFolder.ProgramFilesX86)}/Microsoft Visual Studio/2017/Professional/MSBuild/15.0/Bin/MSBuild.exe";
```

Comment:
MSBuild 15 has moved inside the VS folder, so we can't guarantee it's location any more by simply hardcoding a value.

We'll need to pull in https://github.com/Microsoft/vswhere and figure it out at run time.